resources/js/boot/axios.js

```diff
@@ -2,11 +2,11 @@ import store from "../store/index";
 
 window.axios = require("axios");
 
-const appMode = type =>
+const appMode = (type) =>
     ({
         live: "",
-        test: "http://localhost:8200/api",
-        local: ""
+        test: "http://localhost:8000/api",
```

Comment:
#line 7 - 9,  try to get the value of APP_URL by using laravel's .env file

Reply:
```
let APP_URL = process.env.MIX_APP_URL;
axios.defaults.baseURL = APP_URL;
```